site/content/en/docs/start/_index.md

```diff
@@ -33,7 +66,7 @@ Click on the buttons that describe your target platform. For other architectures
 {{% /quiz_row %}}
 
 {{% quiz_row base="/Linux/x86-64" name="Release type" %}}
-{{% quiz_button option="Stable" %}} {{% quiz_button option="Beta" %}}
+{{% quiz_button option="Stable" %}} {{% quiz_button option="Beta" hide="true" %}}
```

Comment:
This will need to be added to all `Beta` buttons, should be 8 in total

Reply:
You're right! Sorry for that